{
	"id": 156145905,
	"body": "This didn't fix the issue for me. My reproducer above still fails on the second all.bash with\r\n\r\n```\r\n--- FAIL: TestGoTestRaceInstallCgo (0.06s)\r\n\tgo_test.go:257: running testgo [tool -n cgo]\r\n\tgo_test.go:272: standard output:\r\n\tgo_test.go:273: /home/austin/go.tmp/pkg/tool/linux_amd64/cgo\r\n\t\t\r\n\tgo_test.go:257: running testgo [test -race -i runtime/race]\r\n\tgo_test.go:276: standard error:\r\n\tgo_test.go:277: # runtime\r\n\t\t../../runtime/atomic_pointer.go:8: import /home/austin/go.tmp/pkg/linux_amd64_race/runtime/internal/atomic.a: object is [linux amd64 devel +f5722a8 Thu Nov 12 14:26:00 2015 +0000 X:none] expected [linux amd64 devel +d54c356 Thu Nov 12 15:35:48 2015 +0000 X:none]\r\n\t\t\r\n\tgo_test.go:286: go [test -race -i runtime/race] failed unexpectedly: exit status 2\r\nFAIL\r\nFAIL\tcmd/go\t28.969s\r\n```\r\n\r\nwhere f5722a8 is master^ and d54c356 is master.",
	"user": {
		"login": "aclements",
		"id": 2688315,
		"type": "User",
		"site_admin": false
	},
	"created_at": "2015-11-12T15:51:46Z",
	"updated_at": "2015-11-12T15:51:46Z"
}
